src/Support/Utility.php

```diff
@@ -409,6 +411,10 @@ public static function contentstackRequest($stack, $queryObject = '', $type = ''
             $proxy_details = $stack->proxy;
             $timeout = $stack->timeout;
 
+            foreach($plugins as $plugin){
+                $plugin->onRequest($stack, $http, $request_headers);
```

Comment:
Change as per above comment in: https://github.com/contentstack/contentstack-php/pull/31/files#r1100089276